### PR TITLE
Dev > Master - Updating version to 1.5.0

### PR DESCRIPTION
**Pull request by NicholasLancey**
Minor version update. The merge contains:

- Adds z-index to active slide to make items clickable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wtc-gallery-component",
-  "version": "1.4.2",
+  "version": "1.4.3",
```
Comment (liamegan) [inline package.json]: Let's change this to a minor version update instead of a patch.

**Review by liamegan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NicholasLancey) [reply package.json]: Updated, thanks!

Comment (liamegan) [reply package.json]: Sorry, versions cascade. So if you're updating the minor version, the patch version should become `0`

Comment (NicholasLancey) [reply package.json]: Cool - good to know. Updated.

**Review by liamegan — APPROVED** [review]

**Comment by NicholasLancey** [comment]
Merging

**Pull request merged**